lms/djangoapps/courseware/tabs.py

```diff
@@ -66,8 +68,7 @@ def is_enabled(cls, course, user=None):
         """
         The "Home" tab is not shown for the new unified course experience.
         """
-        request = RequestCache.get_current_request()
-        return not waffle.flag_is_active(request, UNIFIED_COURSE_EXPERIENCE_FLAG)
+        return not course_experience_config().is_enabled(UNIFIED_COURSE_EXPERIENCE_FLAG, course.id)
```

Comment:
Note: at lunch I'd suggested that these be called features, but upon reflection I think calling them feature_flags is probably a wiser choice. We don't really know how to fully model a feature yet, and when we do then a feature could point to one or more feature flags.